src/Sylius/Behat/Page/Admin/Order/ShowPage.php

```diff
@@ -329,14 +330,32 @@ public function getPaymentAmount()
      */
     public function getPaymentsCount()
     {
-        $payments = $this->getElement('payments')->findAll('css', '.item');
+        try {
+            $payments = $this->getElement('payments')->findAll('css', '.item');
+        } catch (ElementNotFoundException $exception) {
+            return 0;
+        }
 
         return count($payments);
     }
 
     /**
      * {@inheritdoc}
      */
+    public function getShipmentsCount()
```

Comment:
`public function getShipmentsCount(): int` ?

Reply:
For now we didn't introduce any scalar types in Behat, so I would leave it as it is - I'm sure they'll be provided or at once in the future ;)

Reply:
IMO, we should use scalar types when we're adding sth new in Behat, as I did in my PRs. But of course I agree, that we can provided that at once in the future.

Reply:
I just want to avoid confusion, when some of Behat methods have scalar types and some don't 😄  but let's here others opinion, I don't see any problem in introducing them now.

Reply:
I agree with @GSadee although it's not a blocker :)